changelog/changes/yields-change-proud.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "HTTP request metadata"
+type: feature
+authors: raxyte
+pr: 5295
+---
+
+The `from_http` operator now supports the `metadata_field` option when using the
+server mode to include request metadata into the specified field. The metadata
+has the following schema:
+
+| Field                | Type     | Description                           |
+| :------------------- | :------- | :------------------------------------ |
+| `headers`            | `record` | The request headers.                 |
+| `query`              | `record` | The query parameters of the request.  |
+| `path`               | `string` | The path requested.                   |
+| `fragment`           | `string` | The URI fragment of the request.      |
+| `method`             | `string` | The HTTP method of the request.       |
+| `version`            | `string` | The HTTP version of the request.      |
```

Comment:
Is this table also in the docs? The changelog should *always* be a subset of the docs.